.copier-answers.yml

```diff
@@ -10,13 +11,15 @@ github_enable_makepot: false
 github_enable_stale_action: false
 github_enforce_dev_status_compatibility: false
 include_wkhtmltopdf: false
-odoo_test_flavor: Both
+odoo_test_flavor: OCB
```

Comment:
I intentionally did Both option for this because OCA repos are checked with both Odoo and OCB. I am not against for this change but I prefer both.